client.go

```diff
@@ -924,6 +924,55 @@ func (c *Client) MkdirAll(path string) error {
 	return nil
 }
 
+// DeleteAllResources delete files recursively in the directory and Recursively delete subdirectories.
+// An error will be returned if no file or directory with the specified path exists
+func (c *Client) DeleteAllResources(filePath string) error {
```

Comment:
Name of function and godoc should be modeled on https://pkg.go.dev/os@go1.20.4#RemoveAll to remain as consistent as possible with the `os` package.

The argument should also parallel the use in the other `Remove` functions. This means the signature should be:
```
func (c *Client) RemoveAll(path string) error
```